src/storage/netapp/ontap/restapi/mode/volumes.pm

```diff
@@ -249,11 +297,21 @@ sub manage_selection {
             write_iops    => $_->{metric}->{iops}->{write},
             other_iops    => $_->{metric}->{iops}->{other},
             total_iops    => $_->{metric}->{iops}->{total},
-            read_latency  => $_->{metric}->{latency}->{read} / 1000,
-            write_latency => $_->{metric}->{latency}->{write} / 1000,
-            other_latency => $_->{metric}->{latency}->{other} / 1000,
-            total_latency => $_->{metric}->{latency}->{total} / 1000
+            read_latency  => (defined($_->{metric}->{latency}->{read})) ? ($_->{metric}->{latency}->{read} / 1000) : undef,
```

Comment:
In your mockoon file, latency metrics are under endpoint `storage/aggregates/uuid1`, but you don't seem to query it and the latency metrics don't appear in your tests results. Did I miss something?